core/vdbe.rs

```diff
@@ -704,6 +755,23 @@ impl Program {
                                 state.pc += 1;
                             }
                         }
+                        (OwnedValue::Integer(lhs), OwnedValue::Float(rhs)) => {
+                            if (*lhs as f64) <= *rhs {
+                                state.pc = target_pc;
+                            } else {
+                                state.pc += 1;
+                            }
+                        }
+                        (OwnedValue::Float(lhs), OwnedValue::Integer(rhs)) => {
+                            if *lhs <= (*rhs as f64) {
+                                state.pc = target_pc;
+                            } else {
+                                state.pc += 1;
+                            }
+                        }
+                        (_, OwnedValue::Null) | (OwnedValue::Null, _) => {
+                            state.pc = target_pc;
```

Comment:
I guess these instructions should also have a register for `SQLITE_JUMPIFNULL`, but probably not a thing to concentrate on right now